README.md

```diff
@@ -2,76 +2,32 @@
-
-In case you need to delete a puzzle (perhaps you accidentally created a duplicate), you can do so using the trash icon next to a puzzle name.
-
-When a puzzle is created, a Google Spreadsheet are automatically created for the puzzle. These are linked from the Sheets icon.
-
-### Searching
-
-As a hunt progresses, the number of puzzles can get quite large (especially for a hunt like the MIT Mystery Hunt). To make things more manageable, you can use the "Search" box in the top-right to filter puzzles. The search matches puzzle name, answer, status, or tags. For example, the following shows filtering to just the puzzles that need extraction:
-
-![extraction](https://user-images.githubusercontent.com/1312469/209478179-85516d11-9701-4cc5-b608-f5691cab1d5c.png)
-
-### Metas and Tags
+Cardboard helps teams coordinate solving [puzzle hunts](https://en.wikipedia.org/wiki/Puzzle_hunt). It manages and organizes puzzles, keeps track of team members, integrates with other solving tools, and more.
 
-When adding a new puzzle, if you know it's a meta, you can mark it as such by checking the "Meta?" checkbox. This will automatically create a tag with the same name as the meta puzzle. You can then assign other puzzles as belonging to this meta by using the "Assign Metas" button for a puzzle or adding the meta puzzle tag using the "+" icon in the "Tags" column for a puzzle.
+For help **using Cardboard as a solver**, see [here](user-guide.md).
 
-The main puzzles page is grouped by metas, with puzzles belonging to a meta indented underneath the meta. Each puzzle may belong to multiple metas, in which case it will appear multiple times, once under each meta it belongs to.
+For help **setting Cardboard up for a hunt**, see [here](new-hunt-setup.md).
 
-Meta puzzles may also be assigned to other metas (which might be meta-metas).
+For information about **development**, see the [dev guide](dev-guide.md).
 
-In addition to meta tags for puzzles, users can also tag puzzles as high or low priority, logic or word puzzles, or create new tags.
+### Features
 
-### Google Sheets integration
+* Seamlessly supports (almost) any puzzle hunt structure
+* Create and assign tags to puzzles
+* Quickly search and filter on puzzle names, rounds, and tags
+* Remote-friendly! Real-time updates on how recently each puzzle's sheet was active, and who's currently working on it
 
-When a puzzle is added, a Google Sheet is automatically created for it, linked from the Sheet icon. The puzzle sheet is a clone of a template sheet, which may have some useful formulas pre-populated. Ownership of the puzzle sheets is transferred to the owner of the the template sheet - search `pendingowner:me` to see and accept the transfers.
+### Integrations
 
-### Tools and References
+* Creates a Google Sheet for new puzzles automatically
+* Creates and manages Discord channels for each puzzle
+* TODO: does it do more than just channel management?
```

Comment:
We technically have a Discord bot built-in, but that hasn't been really used / playtested much, so probably not worth advertising here.